ops/jenkins/bfd-cbc-build/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG VERSION_JAVA
+ARG MAVEN_VERSION
+
+FROM maven:${MAVEN_VERSION}-openjdk-${VERSION_JAVA} AS base
+
+# NOTE: necessary duplicate entry
+ARG VERSION_JAVA
+
+COPY jdk${VERSION_JAVA}.toolchains.xml /root/.m2/toolchains.xml
+
+ARG ANSIBLE_VERSION
+
+RUN apt -y update && \
```

Comment:
should use `apt-get` when headless

Reply:
> should use `apt-get` when headless

Is that so? I wasn't aware of the benefits of `apt-get` when unattended/headless. Hmm.

Reply:
Of course you're right. Usage of `apt-get` avoids the warning of unstable CLI interface. Thanks!